Automattic/wp-calypso
# Stats: Restore legend to subscriber section

Related to #75245

## Proposed Changes

<img width="1244" alt="image" src="https://user-images.githubusercontent.com/4044428/230208047-eb15c0eb-7f53-4b80-b586-b1682f46fcaf.png">

* Adds a dynamic chart legend in lieu of tooltips, which were determined to be nontrivial to implement.
* This dynamic legend can be used to selectively render data when multiple series are being rendered. See this [official demo](https://leeoniya.github.io/uPlot/bench/uPlot.html) where you can deselect series from the bottom legend.

## Testing Instructions



* Spin up the live branch for this PR.
* Navigate to the traffic stats page at `/stats/subscribers/:siteSlug` with the query string `?flags=stats/subscribers-section`.
* Ensure that the new chart renders at the bottom of the screen.
* Try resizing the window. Ensure that the chart resizes as expected.
